experimental/dds/attributable-map/src/mapKernel.ts

```diff
@@ -84,8 +80,6 @@ export interface IMapDataObjectSerialized {
 type MapKeyLocalOpMetadata = IMapKeyEditLocalOpMetadata | IMapKeyAddLocalOpMetadata;
 type MapLocalOpMetadata = IMapClearLocalOpMetadata | MapKeyLocalOpMetadata;
 
-/* eslint-disable @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-member-access */
```

Comment:
Do we know why these aren't necessary anymore?

Reply:
I think they were unrelated, but I have format on save turned on and it removed a bunch of lints that were no longer used apparently.